WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift

```diff
@@ -8,7 +9,11 @@ final class WeatherViewController: UIViewController {
     private var landscapeTopConstraint: NSLayoutConstraint = NSLayoutConstraint()
     private var portraitTopConstraint: NSLayoutConstraint = NSLayoutConstraint()
 
-    init () {
+    private let disposeBag = DisposeBag()
+    private let viewModel: WeatherViewModelProtocol
+
+    init (viewModel: WeatherViewModelProtocol) {
```

Comment:
```suggestion
    init(viewModel: WeatherViewModelProtocol) {
```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/0df0b8e6ef1945dbb46c9bd130e52d369b861265)